src/codec/id.rs

```diff
@@ -907,7 +923,7 @@ impl From<AVCodecID> for Id {
 			AV_CODEC_ID_HDMV_TEXT_SUBTITLE => Id::HDMV_TEXT_SUBTITLE,
 
 			/* other specific kind of codecs (generally used for attachments) */
-			AV_CODEC_ID_TTF       => Id::TTF,
+			AVCodecID_AV_CODEC_ID_TTF       => Id::TTF,
```

Comment:
Oh, I see. It should be possible and bindgen builder even has setting for it (https://docs.rs/bindgen/0.22.1/bindgen/struct.Builder.html#method.prepend_enum_name), but it doesn't seem to work as expected in this case. I've tried... It only seems to work, when you set constify_enum. I don't know if this is expected from bindgen side or a bug.